include/openssl/mem.h

```diff
@@ -250,6 +250,9 @@ OPENSSL_EXPORT size_t CRYPTO_secure_used(void);
 // OPENSSL_secure_malloc calls |OPENSSL_malloc|.
 OPENSSL_EXPORT void *OPENSSL_secure_malloc(size_t size);
 
+// OPENSSL_secure_zalloc calls |OPENSSL_zalloc|.
+OPENSSL_EXPORT void *OPENSSL_secure_zalloc(size_t size);
+
 // OPENSSL_secure_clear_free calls |OPENSSL_clear_free|.
 OPENSSL_EXPORT void OPENSSL_secure_clear_free(void *ptr, size_t len);
```

Comment:
Adding some documentation here on what secure heaps are and that we don't support this yet would help.